### DISCUSSION TITLE
mac-os builds fail randomly

### DISCUSSION THREAD

**Discussion by ip7e** [community/community#25139] [Actions]
<p>Hi all, </p>
<p>We’re experiencing problems with mac-os electron builds.</p>
<p>It fails randomly on different states on every other run with exit code 1.</p>
<p>We’ve tried to revert all changes back to it’s original, for which we’ve had a successful build and run it again, but it fails anyway. </p>
<p>Nothing has been changed in the workflow file as well.</p>
<p>We’ve also tried to build it locally within a similar environment (mac-os 10.15.4, node 12.16.1) successfully.</p>
<p>Do you have any suggestions for troubleshooting?</p>
<p>Thanks</p>

**Comment by ip7e** [comment, marked as answer]
<p>i think it’s related to the nodejs hiccup<br>
<a href="https://github.com/nodejs/build/issues/2263" rel="nofollow noopener">https://github.com/nodejs/build/issues/2263</a></p>

**Comment by weide-zhou** [comment]
<p>Hi <a class="mention" href="/ddramone">@ddramone</a> ,</p>
<p>The referred link is closed, please check and confirm whether the issue is gone for your workflow. </p>
<p>If it persists, please share your repo/sample repo for further investigation.</p>
<p>Thanks.</p>

**Comment by ip7e** [comment]
<p>I can confirm it was related and now everything works as expected. thanks!</p>
